ddl/reorg_test.go

```diff
@@ -70,6 +70,8 @@ func (s *testDDLSuite) TestReorg(c *C) {
 	c.Assert(err, NotNil)
 
 	<-done
+	// Make sure the function of f is returned.
+	time.Sleep(5 * time.Millisecond)
```

Comment:
With the function really out there are still a little time. So it is only occasionally go wrong.